Fixed generate-docs to run only the specified commands (#1211)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Added a validation that layoutscontainer's id and name are matching. Updated the format of layoutcontainer to include update_id too.
* Added a validation that commands' names and arguments in core packs, or scripts' arguments do not contain the word incident.

> * Fixed issue where **generate-docs** with -c flag runs all the commands and not just the commands specified in the -c flag.

Reviewer: ```suggestion
* Fixed issue where running the **generate-docs** command with -c flag ran all the commands and not just the commands specified by the flag.
```
